plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -620,13 +622,14 @@ createWidget("discourse-poll-pie-chart", {
     }
     contents.push(btn);
     contents.push(chart);
+    contents.push(h(`div#poll-results-legend-${attrs.id}.pie-chart-legends`));
     return contents;
   }
 });
 
 function pieChartConfig(data, labels, opts = {}) {
-  const aspectRatio = "aspectRatio" in opts ? opts.aspectRatio : 2.0;
-  const displayLegend = "displayLegend" in opts ? opts.displayLegend : true;
+  const aspectRatio = "aspectRatio" in opts ? opts.aspectRatio : 2.2;
+  const strippedLabels = labels.map(l => l.replace(/<[^>]*>?/gm, "").trim());
```

Comment:
Stripping HTML can be error prone. I just want to confirm that there's no other way to have the chart library omit this?

Reply:
@eviltrout Chartjs does not have a strip out html feature.

I could do something more robust like this
```
function strip(html){
   var doc = new DOMParser().parseFromString(html, 'text/html');
   return doc.body.textContent || "";
}
```

Reply:
Sure, try it out but be sure to check performance and if it works in IE11.

Reply:
@eviltrout Do you trust caniuse? It says that it has full support for every major browser.

https://caniuse.com/#feat=mdn-api_domparser_parsefromstring_html